### PR TITLE
CATROID-516 Remove Redundant "if" statement

### PR DESCRIPTION
**Pull request by Abdelaty**
Signed-off-by: abdelaty <abdelaty.mohammedmagdi@gmail.com>

Remove Redundant "if" statement

- [x] Include the name of the Jira ticket in the PR’s title
- [ ] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/nxt/sensors/NXTI2CUltraSonicSensor.java [hunk 1]
```diff
@@ -38,8 +38,8 @@
 	private static final int DEFAULT_VALUE = 255;
 	private static final String TAG = NXTI2CUltraSonicSensor.class.getSimpleName();
 
-	private enum DistanceUnit {
-		DUMMY, CENTIMETER, INCH;
```
Comment (wslany) [inline catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/nxt/sensors/NXTI2CUltraSonicSensor.java]: PLEASE EXPLAIN: Why did you move this down?

Comment (Abdelaty) [reply catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/nxt/sensors/NXTI2CUltraSonicSensor.java]: I think that when I checked the refactor and reformat code option. it changed the place of the method 
image
I will move it now. Thanks
